exs/mundo_1/javascript/029.js

```diff
@@ -0,0 +1,14 @@
+const input = require('readline-sync').question //Chamando o modulo readline-sync para entrada de dados
+
+//Entrada de dados 
+let velo = input("Velocidade do carro: ")
+
+// Verificando se a velocidade esta acima do limte com if e else
```

Comment:
Correção ortográfica necessária aqui também

```suggestion
// Verificando se a velocidade esta acima do limite com if e else
```